spec/services/appointment_notification_service_spec.rb

```diff
@@ -68,13 +64,11 @@
       )
 
       AppointmentNotificationService.send_after_missed_visit(appointments: appointments)
-
-      ENV["ENABLE_WHATSAPP_APPOINTMENT_REMINDERS"] = whatsapp_appointment_reminders
     end
 
-    context "if WHATSAPP_APPOINTMENT_REMINDERS feature is disabled" do
+    context "if whatsapp_appointment_reminders feature is disabled" do
       it "should skip sending reminders for appointments for which SMS reminders are already sent" do
-        expect(FeatureToggle).to receive(:enabled?).with("WHATSAPP_APPOINTMENT_REMINDERS").and_return(false)
+        Flipper.disable(:whatsapp_appointment_reminders)
```

Comment:
Since Flipper's using a DB store, any feature flag changes inside the spec are rolled back automatically at the end of the spec?

Reply:
Yup that's correct.